webapp/packages/plugin-authentication-administration/src/Administration/Users/UsersTable/User.tsx

```diff
@@ -63,8 +63,9 @@ export const User = observer<Props>(function User({ user, displayAuthRole, isMan
       <TableColumnValue title={teams} ellipsis>
         {teams}
       </TableColumnValue>
-      <TableColumnValue>
+      <TableColumnValue centerContent>
         <Checkbox
+          className={style['checkbox']}
```

Comment:
It's incorrect, we are changing the display and other "inner" styles of the checkbox itself.
The proper way is to add a container within the checkbox component and apply styles to that container instead.

Reply:
moreover this styles can be applyed with tailwind classes